README.md

```diff
@@ -139,16 +139,31 @@ May be called eg. in the `componentDidMount` of your main component. This method
 To see how to handle errors read [`signIn()` method](#signin)
 
 ```js
-getCurrentUser = async () => {
+getCurrentUserInfo = async () => {
   try {
     const userInfo = await GoogleSignin.signInSilently();
     this.setState({ userInfo });
   } catch (error) {
-    console.error(error);
+    if (error.code === statusCodes.SIGN_IN_REQUIRED) {
+      // user has not signed in yet
+    } else {
+      // some other error
+    }
   }
 };
 ```
 
+#### `isSignedIn()`
+
+This method may be used to find out whether some user is currently signed in. It returns a promise which resolves with a boolean value (it never rejects). In the native layer, this is a synchronous call.
+
```

Comment:
I would also mention a scenario where a call to the `isSignedIn()` is probably the most useful: when the app is offline. Note: I didn't test that it actually works like that.

Reply:
Actually tested this on iOS simulator and it works offline.

Reply:
should be clear now